python/using_for_in_python_command_line.md

```diff
@@ -0,0 +1,15 @@
+# Using map function instead of for loop in Python command line option
+
+In this sceanrio, we will write a for loop for iterating over range using 
+Python command prompt
```

Comment:
How about using more of a "Here is something I learned" tone rather than "Let me show you something I know"